### DISCUSSION TITLE
How to use undici's FormData to send FormData in a (POST) request?

### DISCUSSION THREAD

**Discussion by MxHex** [nodejs/undici#1938] [Q&A] [answered]
Hey folks, greetings 👋 
In my typescript project I want to use undici to send **FormData** but I really can't find any how-to how to do this with undici.
Undici does export a class `FormData` which doesn't have a constructor though! So the error I get is the following:

> TypeError: undici_1.FormData is not a constructor

My example code goes something like this:

```
import { Dispatcher, request, FormData } from 'undici';

try {
    const form = new FormData();
    const options: Dispatcher.RequestOptions = {
        path: "/path/to/resource",
        method: "POST",
        body: form,
        throwOnError: true
    }
    await request("https://myhost.com", options);
} catch (error) {}
```
How else would I construct a FormData object that undici allows?
Help would be very much appreciated!  🙏

**Comment by ronag** [comment]
Sounds like a bug. PR welcome!

**Reply by MxHex** [reply]
PR created: https://github.com/nodejs/undici/pull/1939 - Review welcome :)

**Comment by KhafraDev** [comment, marked as answer]
this was fixed
